doc/snmp/snmp-configdb-migration-hld.md

```diff
@@ -0,0 +1,221 @@
+  location   show runningconfiguration snmp location
+  users      show runningconfiguration snmp users
+admin@str-s6000-acs-11:~$ 
+```
+
+show run snmp community
+```
+admin@str-s6000-acs-11:~$ show run snmp community
+Community String    Community Type
+------------------  ----------------
+Qi                  RO
+Travis              RO
+Bill                RO
+Jack                RW
+public              RO
+Joker               RW
+admin@str-s6000-acs-11:~$ 
+```
+
+show run snmp contact
+```
+admin@str-s6000-acs-11:~$ show run snmp contact
+Contact    Contact Email
+---------  -----------------
+Joe        joe@microsoft.com
+admin@str-s6000-acs-11:~$ 
+```
+
+show run snmp location
+```
+admin@str-s6000-acs-11:~$ show run snmp location 
+Location
+----------
+Redmond
+admin@str-s6000-acs-11:~$ 
+```
+
+show run snmp users
+```
+admin@str-s6000-acs-11:~$ show run snmp users
+User    Type    Auth Type    Auth Password    Encryption Type    Encryption Password
+------  ------  -----------  ---------------  -----------------  ---------------------
+Travis  Priv    SHA          TravisAuthPass   AES                TravisEncryptPass
+admin@str-s6000-acs-11:~$
+```
+
+
+# Config Commands
+```
+admin@str-s6000-acs-11:~$ sudo config snmp -h
+Usage: config snmp [OPTIONS] COMMAND [ARGS]...
+
+  SNMP configuration tasks
+
+Options:
+  -?, -h, --help  Show this message and exit.
+
+Commands:
+  community
+  contact
+  location
+  user
+admin@str-s6000-acs-11:~$ 
+```
+
+sudo config snmp community 
+```
+admin@str-s6000-acs-11:~$ sudo config snmp community -h
+Usage: config snmp community [OPTIONS] COMMAND [ARGS]...
+
+Options:
+  -?, -h, --help  Show this message and exit.
+
+Commands:
+  add  Add snmp community
+  del  Add snmp community
+admin@str-s6000-acs-11:~$ 
+```
+
+sudo config snmp contact  
+```
+admin@str-s6000-acs-11:~$ sudo config snmp contact -h
+Usage: config snmp contact [OPTIONS] COMMAND [ARGS]...
+
+Options:
+  -?, -h, --help  Show this message and exit.
+
+Commands:
+  modify  Modify snmp contact
+admin@str-s6000-acs-11:~$ 
+```
+
+sudo config snmp location 
+```
+admin@str-s6000-acs-11:~$ sudo config snmp location -h
+Usage: config snmp location [OPTIONS] COMMAND [ARGS]...
+
+Options:
+  -?, -h, --help  Show this message and exit.
+
+Commands:
+  modify  Modify snmp location
+admin@str-s6000-acs-11:~$ 
+```
+
+sudo config snmp user 
+```
+admin@str-s6000-acs-11:~$ sudo config snmp user -h
+Usage: config snmp user [OPTIONS] COMMAND [ARGS]...
+
+Options:
+  -?, -h, --help  Show this message and exit.
+
+Commands:
+  add  Add snmp user
+  del  Add snmp user
+admin@str-s6000-acs-11:~$ 
+```
+
+# Migration Plan
+In order to move from using the snmp.yml file to using the Redis ConfigDB here are the things the need to be done to move this in a way that is backward compatible. 
+1. Create a python conversion script to parse the data in snmp.yml file and store it in ConfigDB using the above schema.
```

Comment:
The SNMP information will be pulled from both configDB and snmp.yml. Why you still need a conversion script?

Reply:
If we pull the info from the snmp.yml file into the configdb whenever the docker-snmp container is restarted it makes it much cleaner for show/config commands and allows for backward compatibility.  This will be much cleaner.  I've discussed this with Suvarna and Joe but didn't get a chance to run it by you yet.